cloud/libvirt/actuators/machine/utils/domain.go

```diff
@@ -467,19 +457,28 @@ func CreateDomain(name, ignKey, volumeName, hostName, networkInterfaceName, netw
 
 	log.Printf("[INFO] setCoreOSIgnition")
 	if ignKey != "" {
-		if err := setCoreOSIgnition(&domainDef, ignKey); err != nil {
+		ignVolume, err := getVolumeFromPool(ignKey, poolName, virConn)
+		if err != nil {
+			return fmt.Errorf("error getting ignition volume: %v", err)
```

Comment:
Add which pool:

    error getting ignition from pool %q ...

It will help any subsequent debugging if we know which pool (by name).

Reply:
> Add which pool...

I'd rather handle that in `getVolumeFromPool`, where all callers can benefit.  And at least [one][1] of the errors there already includes the pool name.

[1]: https://github.com/openshift/cluster-api-provider-libvirt/blob/00b16f152ee24782eb6cdf36d8ca988258d46e49/cloud/libvirt/actuators/machine/utils/volume.go#L200